diff --git a/node_modules/react-native/React/Views/RCTTVView.m b/node_modules/react-native/React/Views/RCTTVView.m
index 4f36852..a382fbc 100644
--- a/node_modules/react-native/React/Views/RCTTVView.m
+++ b/node_modules/react-native/React/Views/RCTTVView.m
@@ -35,7 +35,7 @@
         @"shiftDistanceY" : @2.0f,
         @"tiltAngle" : @0.05f,
         @"magnification" : @1.0f,
-        @"pressMagnification" : @1.0f,
+        @"pressMagnification" : @1.1f,
         @"pressDuration" : @0.3f,
         @"pressDelay" : @0.0f
       };
diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
new file mode 100644
index 0000000..361f5fb
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.env
@@ -0,0 +1 @@
+export RCT_METRO_PORT=8081
