diff --git a/node_modules/react-native-paper/src/components/Button.tsx b/node_modules/react-native-paper/src/components/Button.tsx
index cca903d..32ad4ff 100644
--- a/node_modules/react-native-paper/src/components/Button.tsx
+++ b/node_modules/react-native-paper/src/components/Button.tsx
@@ -87,6 +87,8 @@ type Props = React.ComponentProps<typeof Surface> & {
    * testID to be used on tests.
    */
   testID?: string;
+  onFocus?: () => void;
+  onBlur?: () => void;
 };
 
 /**
@@ -134,6 +136,8 @@ const Button = ({
   accessibilityLabel,
   onPress,
   onLongPress,
+  onFocus,
+  onBlur,
   style,
   theme,
   contentStyle,
@@ -269,6 +273,8 @@ const Button = ({
         delayPressIn={0}
         onPress={onPress}
         onLongPress={onLongPress}
+        onFocus={onFocus}
+        onBlur={onBlur}
         onPressIn={handlePressIn}
         onPressOut={handlePressOut}
         accessibilityLabel={accessibilityLabel}
