diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index e9cebf4..2bbcebe 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -100,6 +100,7 @@ def REACT_NATIVE_DIR = resolveReactNativeDirectory()
 def reactProperties = new Properties()
 file("$REACT_NATIVE_DIR/ReactAndroid/gradle.properties").withInputStream { reactProperties.load(it) }
 
+def REACT_NATIVE_GROUP = reactProperties.getProperty("GROUP")
 def REACT_NATIVE_VERSION = reactProperties.getProperty("VERSION_NAME")
 def REACT_NATIVE_MINOR_VERSION = REACT_NATIVE_VERSION.startsWith("0.0.0-") ? 1000 : REACT_NATIVE_VERSION.split("\\.")[1].toInteger()
 
@@ -258,7 +259,7 @@ def kotlin_version = safeExtGet('kotlinVersion', project.properties['RNGH_kotlin
 dependencies {
     //noinspection GradleDynamicVersion
     if (REACT_NATIVE_MINOR_VERSION >= 71) {
-        implementation "com.facebook.react:react-android" // version substituted by RNGP
+        implementation "${REACT_NATIVE_GROUP}:react-android:${REACT_NATIVE_VERSION}" // version substituted by RNGP
     } else {
         implementation 'com.facebook.react:react-native:+' // from node_modules
     }
