diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index 6ca84ac..43a64ca 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -204,6 +204,7 @@ def reactNativeRootDir = resolveReactNativeDirectory()
 def reactProperties = new Properties()
 file("$reactNativeRootDir/ReactAndroid/gradle.properties").withInputStream { reactProperties.load(it) }
 
+def REACT_NATIVE_GROUP = reactProperties.getProperty("GROUP")
 def REACT_NATIVE_VERSION = reactProperties.getProperty("VERSION_NAME")
 def REACT_NATIVE_MINOR_VERSION = REACT_NATIVE_VERSION.startsWith("0.0.0-") ? 1000 : REACT_NATIVE_VERSION.split("\\.")[1].toInteger()
 def REANIMATED_PACKAGE_BUILD = System.getenv("REANIMATED_PACKAGE_BUILD")
@@ -444,7 +445,7 @@ android {
         }
     }
     defaultConfig {
-        minSdkVersion safeExtGet("minSdkVersion", 16)
+        minSdkVersion safeExtGet("minSdkVersion", 21)
         targetSdkVersion safeExtGet("targetSdkVersion", 30)
         versionCode 1
         versionName "1.0"
@@ -1034,9 +1035,9 @@ dependencies {
     implementation "androidx.core:core:1.6.0"
 
     if (REACT_NATIVE_MINOR_VERSION >= 71) {
-        implementation "com.facebook.react:react-android" // version substituted by RNGP
+        implementation "${REACT_NATIVE_GROUP}:react-android:${REACT_NATIVE_VERSION}" // version substituted by RNGP
         if (JS_RUNTIME == "hermes") {
-            implementation "com.facebook.react:hermes-android" // version substituted by RNGP
+            implementation "${REACT_NATIVE_GROUP}:hermes-android:${REACT_NATIVE_VERSION}" // version substituted by RNGP
         }
     } else {
         implementation "com.facebook.react:react-native:+" // From node_modules
